apps/backend/src/controllers/neighborhoods.ts

```diff
@@ -3,95 +3,102 @@ import { Neighborhood } from '@prisma/client';
 import catchError from '../utils/catchError';
 import prismaClient from '../../prismaClient';
 import middleware from '../utils/middleware';
-import routeHelpers from '../utils/routeHelpers';
-import { CustomRequest } from '../types';
+import {
+  CreateNeighborhoodData, NeighborhoodWithRelatedFields, RequestWithAuthentication,
+} from '../types';
+import neighborhoodServices from '../services/neighborhoodServices';
 
 const neighborhoodsRouter = express.Router();
 
 neighborhoodsRouter.get('/', catchError(async (_req: Request, res: Response) => {
-  const neighborhoods = await prismaClient.neighborhood.findMany({});
-  if (neighborhoods.length === 0) {
-    res.status(404).end();
-  } else {
-    res.send(neighborhoods);
-  }
+  const neighborhoods = await neighborhoodServices.getAllNeighborhoods();
+  return res.status(200).send(neighborhoods).end();
```

Comment:
Yupp. However, lint is forcing me to include an explicit `return` in some of the routes. See if you can remove `return` without lint generating a warning.

Reply:
Yep, you're right. Let's leave it like that.

Reply:
I wanna chime in to suggest we adopt a uniform style, i.e. return all responses or don't (personally, I prefer returning over `if/else` statements).

Reply:
I have used `return` inside the guard clause only.

Otherwise, `res.send(...)` is the very last statement in the route definition. So it is implied that function will return immediately after that.